oak/server/rust/oak_runtime/src/runtime/tests.rs

```diff
@@ -49,13 +62,25 @@ fn run_node_body(node_label: &Label, node_body: Box<NodeBody>) {
         .node_configure_instance(new_node_id, "test_module.test_function", &node_label);
 
     let node_instance = TestNode { node_body };
-    let result = proxy.runtime.node_start_instance(
-        &new_node_name,
-        Box::new(node_instance),
-        test_proxy,
-        oak_abi::INVALID_HANDLE,
-    );
-    assert_eq!(Ok(()), result);
+    info!("Start test Node instance");
+    let join_handle = proxy
+        .runtime
+        .node_start_instance(
+            &new_node_name,
+            Box::new(node_instance),
+            test_proxy,
+            oak_abi::INVALID_HANDLE,
+        )
+        .expect("failed to start test Node");
+
+    // Wait for test Node execution to complete before terminating,
+    // so that any ABI functions invoked by the test Node don't just
+    // return `ErrTerminated`.
+    let _ = join_handle.join();
```

Comment:
Perhaps unwrap / expect this, otherwise panics may be ignored (I think?)

Reply:
Good point, done &ndash; and this might actually fix #905 (previously the test thread was never joined).